topics/reference_guide/sdk_setup_assistance.md

```diff
@@ -0,0 +1,35 @@
+[//]: # (title: Assisting in Setting Up an SDK)
+
+<!-- Copyright 2000-2020 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
+
+Prompting the user with a notification to set up an SDK can help them get up-and-running with a plugin faster. The IntelliJ Platform offers extension point `com.intellij.projectSdkSetupValidator`, where you can register an implementation of [`ProjectSdkSetupValidator`](upsource:///platform/lang-impl/src/com/intellij/codeInsight/daemon/ProjectSdkSetupValidator.java) to notify the user if they are missing an SDK.
```

Comment:
I'd add an article here: "The IntelliJ Platform offers **the** extension point ..."

Unless, of course, you are a native speaker and you know better :)

Reply:
Lol I am a native speaker, but I still racked my brain for 10 minutes thinking whether the original sentence was incorrect. I think putting "the" is more correct though, since a similar sentence like "The restaurant offers {} menu option 'escargot'..." wouldn't make sense without "the" at {}. (English is the weirdest language).

I'll push up a change now to fix that.